CHANGELOG.md

```diff
@@ -6,7 +6,8 @@
 
 ### Breaking Changes
 
-_None_
+* This is a test. 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGenPlugin/blob/6fcc6b0e593456b63d45396fd76d26550125e5f9/CHANGELOG.md#L9"></span>Line describing your entry should end with a period and 2 spaces.</td>
    </tr>
  </tbody>
</table>